buildSrc/build.gradle

```diff
@@ -56,7 +56,6 @@ allprojects {
 
 dependencies {
     api(project("plugins"))
-    implementation project(':public')
```

Comment:
This was added in the fork-only to have access to classes from this module. Now these classes aren't used, and we can revert it